src/loaders/revealjs-loader.js

```diff
@@ -1,14 +1,18 @@
-const querystring = require("querystring");
 const { marked } = require("marked");
 const Prism = require("prismjs");
 require("prismjs/components/")();
 
 module.exports = function (content) {
-  const loaderOptions = querystring.parse(this.resourceQuery.substring(1));
+  const resourceQuery = new URLSearchParams(this.resourceQuery.substring(1));
+  const options = this.getOptions();
+  if (resourceQuery.has("titleOnly")) {
+    return content.match(/^[^\S\r\n]*#[^\S\r\n]*([^#\r\n]+)$/m)?.[1] || "";
```

Comment:
Will miss HTML `<h1>` titles.

Reply:
Right. I'm OK with this limitation for now. What do you think?

Reply:
Right. I'm OK with this limitation for now. What do you think?

Reply:
I don't think any training is using another form than `#`, but the missing title message as to not propose `<h1>` anymore.